fbw-a380x/src/systems/systems-host/index.ts

```diff
@@ -118,6 +121,9 @@ class SystemsHost extends BaseInstrument {
     this.gpws = new LegacyGpws(this.soundManager);
     this.gpws.init();
 
+    this.tcas = new LegacyTcasComputer(this.soundManager);
+    this.tcas.init();
```

Comment:
Implement `Instrument` (i.e. rename update to onUpdate and add `implements Instrument`), and just put it on the backplane. ☺️

Reply:
This is a nice idea, but at best I can get rid of the TcasComponent definition, I can't actually refactor it into backplane as there's a dependency on legacy sound manager.